mpt/impl/smtp/cassandra-rabbitmq-object-storage/src/test/resources/imapserver.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
```

Comment:
My bet is that we don't need IMAP, LMTP, POP3, etc... while testing SMTP. Maybe we can omit unneeded files?

Reply:
I tried !  If absent, the configuration mechanism will pick up the default configs in sever/protocols and try to bind on priviledged ports (such as 143), somehow despite having test-jar dependencies to the guice-cassandra and guice-cassandra-rabbit project it did not find the files there. I gave up after wasting a full day trying to trace it :(

Reply:
Ok. I though the default was disabled.

Maybe instead we can pass an empty configuration block?

Reply:
>Ok. I though the default was disabled.
It definitely isn't for IMAP I didn't try for the others

>Maybe instead we can pass an empty configuration block?
The idea never crossed my ind but it does work rather well and is much more satisfying ! fixedup